docs/src/doc/create-reusable-components/02_designing_components.stories.mdx

```diff
@@ -178,32 +195,29 @@ Although unlikely, it may happen that the newly needed alias token does not yet
-  - **Responsive** behavior specifications are also outlined
-
-- Basic component documentation (as defined in 1.4):
-
+- Component specs that are fully outlined using tokens (as defined in 1.4):<br/>
+  - **Visual** specification (sizes, paddings, typographic styles, colors) are fully defined using design tokens ([See 2.2](#22-using-tokens-to-specify-a-new-component)).
+  - **States**’ specifications (e.g. default, hover, active, focus, error, disabled) and **interactive behavior** are defined and fully specified. In both cases, the user actions leading to each state should also be indicated.
+  - **Responsive** behavior specifications are also outlined <br/>
+<br/>
+- Basic component documentation (as defined in 1.4):<br/>
   - A distinctive **name** for the component and its relevant typologies
-  - A **short description** of the overall purpose of the component
-
-- Storybook indications:
-
+  - A **short description** of the overall purpose of the component<br/>
+  <br/>
+- Storybook indications:<br/>
   - Specify where, within the current **Storybook structure**, the component should  be classified (new dedicated section or subsection), based on the relationship to its component family (this relationship should also be expressed by the name).
-  - Indicate the required Storybook **knobs**: these are sets of modifiers that allow users to check the visual and interactive properties of a component (e.g change the text of a button label to see its width adjust).
-
-- **Assets** used by components (e.g. icons) should be included in Figma, but ideally these could be exported in an svg format and attached to the component story in Phabricator, to make them directly available for download
+  - Indicate the required Storybook **knobs**, if needed: these are sets of modifiers that allow users to check the visual and interactive properties of a component.<br/>
+  <br/>
+- **Assets** used by components (e.g. icons) should be included in Figma, but ideally these could be exported in an svg format and attached to the component story in Phabricator, to make them directly available for download. <br/>
 
 👉🏻  Designers should dedicate time to go through handover files with the developers in their hike in order to make sure that the documentation is intelligible and well structured for everyone, and to detect potential missing requirements before moving into implementation.
 
 ******
 
-_Last update: 14/08/2020. Does this file need corrections or an update? Feel free to [edit it on GitHub](https://github.com/wmde/wikit/blob/master/docs/src/doc/create-reusable-components/02_designing_components.stories.mdx)_
+_Last update: 19/08/2020, by [Sarai Sánchez](https://meta.wikimedia.org/wiki/User:SaraiSan-WMDE). Does this file need corrections or an update? Feel free to [edit it on GitHub](https://github.com/wmde/wikit/blob/master/docs/src/doc/create-reusable-components/02_designing_components.stories.mdx)_
```

Comment:
Is this whole message here on purpose or was it generated by some tool? 🤔

Reply:
It's manually introduced, so it may be easily become outdated. It may be worth eliminating the date and author, and simply leave the encouragement line :)

Reply:
oops. Just noticed we overlapped. I didn't see the other comment!